src/renderer/core/layout/store/layoutStore.ts

```diff
@@ -348,6 +351,20 @@ class LayoutStoreImpl implements LayoutStore {
     return computed(() => this.version)
   }
 
+  /**
+   * Get Vue node dragging state for selection toolbox
+   */
+  getVueNodeDraggingState(): Ref<boolean> {
+    return this.isDraggingVueNodes
+  }
```

Comment:
Why have a getter and setter instead of exposing the ref?

Reply:
It's a [store](https://pinia.vuejs.org/core-concepts/#Setup-Stores) though, right?

Reply:
If we expose the raw ref in the getter anyway, why not just make isDraggingVueNodes a public readonly property?

Reply:
How is it any different than the setter here?